src/locales/cy/translation.json

```diff
@@ -199,6 +199,24 @@
         }
       }
     },
+    "pagePersistIdentity": {
+      "title": "Rydych wedi profi pwy ydych chi yn barod",
+      "header": "Rydych wedi profi pwy ydych chi yn barod",
+      "content": {
+        "paragraph1": "Rydych wedi brofi pwy ydych chi pan wnaethoch ddefnyddio eich cyfrif GOV.UK o'r blaen. Nid oes angen i chi wneud hyn eto i ddefnyddio'r gwasanaeth hwn.",
```

Comment:
We can use ’ (typographical quotes) here - the Welsh translation people don’t put them in but they are more correct than ' quotes apparently.